package.json

```diff
@@ -21,9 +21,7 @@
   ],
   "author": "nlt2390",
   "dependencies": {
-    "prop-types": "15.6.0",
-    "react": "16.0.0",
-    "react-native": "0.50.1"
+    "prop-types": "^15.0.0",
   },
   "peerDependencies": {
     "react": "*",
```

Comment:
can u add prop-types here too ?

Reply:
Yep, they say you should add it there in the read me: https://github.com/facebook/prop-types#how-to-depend-on-this-package

Reply:
Sorry for the delayed respose

Reply:
I mean, plz make a new commit adding it to peerDependencies and I will merge.